src/Init/Data/String/Basic.lean

```diff
@@ -24,23 +24,41 @@ instance : LT String :=
 instance decLt (s₁ s₂ : @& String) : Decidable (s₁ < s₂) :=
   List.hasDecidableLt s₁.data s₂.data
 
+/--
+Returns the length of a string.
+* `"".length = 0`
+* `"abc".length = 3`
```

Comment:
Generally, bulleted lists of examples should be a separate paragraph and have some indication that this is what they are, eg:

```suggestion

Examples:
* `"".length = 0`
* `"abc".length = 3`
```

In this particular case, something that shows that it's code points, rather than bytes or grapheme clusters, would be useful, but not essential.

Reply:
I added `"L∃∀N".length = 4` to differentiate between code points and bytes.